packages/purpose-outbound-writer/.env

```
@@ -0,0 +1,14 @@
+LOG_LEVEL=info
+
+KAFKA_CLIENT_ID="purpose"
+KAFKA_GROUP_ID="purpose-group"
```

minor, but in case of local testing on deployed cluster, it's easier to understand the state
```suggestion
KAFKA_GROUP_ID="purpose-group-local"
```